homeassistant/components/eafm/sensor.py

```diff
@@ -63,6 +73,8 @@ async def async_update_data():
         # find themselves.
         data["measures"] = {measure["@id"]: measure for measure in measures}
 
+        data["stageScale"] = {0: stage_scale for stage_scale in stage_scales}
```

Comment:
If there were multiple stage scales, this key will always be 0. So this picks the last one. In that case why bother making a mapping at all?